src/data/quot.lean

```diff
@@ -404,6 +404,13 @@ protected lemma induction_on₃' {p : quotient s₁ → quotient s₂ → quotie
   (h : ∀ a₁ a₂ a₃, p (quotient.mk' a₁) (quotient.mk' a₂) (quotient.mk' a₃)) : p q₁ q₂ q₃ :=
 quotient.induction_on₃ q₁ q₂ q₃ h
 
+/-- A version of `quotient.rec_on_subsingleton` taking `{s₁ : setoid α}` as implicit arguments
+instead of instance arguments. -/
```

Comment:
```suggestion
/-- A version of `quotient.rec_on_subsingleton` taking `{s₁ : setoid α}` as an implicit argument
instead of an instance argument. -/
```